src/main/java/dev/jbang/cli/Info.java

```diff
@@ -94,6 +97,8 @@ static class ScriptInfo {
 		String description;
 		String gav;
 		String module;
+		String docs;
```

Comment:
The type of this shouldn't be a `String` but a `ResourceRef`. And once that is done you don't need `rootFile`.